programs/leap-cli11/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_library(leap-cli11 INTERFACE)
+target_include_directories(leap-cli11 INTERFACE include)
```

Comment:
since `leap-cli11` is just a library, would it be a better fit in `libraries/` instead of `programs/`?

Reply:
Totally makes sense, moving... Thank you!